circle.yml

```diff
@@ -0,0 +1,14 @@
+dependencies:
+  cache_directories:
+    - "~/.stack"
+  pre:
```

Comment:
Will this run every time, or will it get cached between builds? If it runs every time, do we want to download this into a directory that gets cached?